Trustroots/trustroots
# Rewrite email confirmed landing page into React component

#### Proposed Changes

* Rewrite email confirmed landing page into React component
* Simplify styles for this page; less code, and text is now more readable/accessible


**Before**

<img width="855" alt="Screenshot 2021-02-28 at 17 19 51" src="https://user-images.githubusercontent.com/87168/109423986-6e9ca580-79ea-11eb-9746-429b21e2c7a5.png">

**After**

<img width="376" alt="Screenshot 2021-02-28 at 17 20 16" src="https://user-images.githubusercontent.com/87168/109423989-72302c80-79ea-11eb-83bd-b397c524bdf6.png">


<img width="909" alt="Screenshot 2021-02-28 at 17 19 33" src="https://user-images.githubusercontent.com/87168/109423982-6ba1b500-79ea-11eb-9008-7e456e7bcac1.png">

#### Testing Instructions

* Just open `/welcome` and check different screen widths. Users land on this page after confirming their email.

Part of react rewrite, and will bring these strings to translators 🎉
